evaluation.py

```diff
@@ -35,7 +37,18 @@ def predict_heatmaps(self, h, X):
         X = np.expand_dims(X, axis=0) # add "batch" dimension of 1 because predict needs shape (1, 256, 256, 3)
         predict_heatmaps = h.model.predict(X)
         predict_heatmaps = np.array(predict_heatmaps) # output shape is (num_hg_blocks, 1, 64, 64, 17)
-        return predict_heatmaps
+
+        print('model prediction metrics: ')
+        predict_mean = np.mean(predict_heatmaps)
+        predict_max = np.max(predict_heatmaps)
+        predict_min = np.min(predict_heatmaps)
+        predict_var = np.var(predict_heatmaps.flatten())
+        print('Mean: {:0.6e}\t Max: {:e}\t Min: {:e}\t Variance: {:e}'.format(predict_mean, predict_max, predict_min, predict_var))
+        normalized_heatmaps = predict_heatmaps / predict_max
+        normalized_heatmaps = normalized_heatmaps - predict_min
+        normalized_heatmaps = predict_heatmaps / np.max(predict_heatmaps)
+
+        return normalized_heatmaps
```

Comment:
Do the normalized predicted heatmaps look different or is this just to normalize the np array?

Reply:
this is to normalize the array. We're getting some major issues with large numbers in the network right now so this is to try to combat that in order to plot it.

Reply:
do you remember the really odd looking heat map? So normally our network output is ~0 to ~1, but occasionally it outputs like 6.0e16 and -6.0e16 (which results in that really blurry image we saw)

Reply:
oh right sweet so this should keep it between the normal output

Reply:
yep exactly. I have to fix our model too so we dont even get to this state